_posts/2024-12-01-18.1.0-release.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Apache Arrow 18.1.0 Release"
+date: "2024-12-01 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 18.1.0 release. This
+covers roughly one month of development work and includes [**15 resolved
+issues**][1] on [**25 distinct commits**][2] from [**8 distinct
+contributors**][2]. See the [Install Page](https://arrow.apache.org/install/) to
+learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected
+highlights of the release. Many other bugfixes and improvements have been made:
+we refer you to the [complete changelog][3].
+
+## Community
+
+Since the 18.0.0 release, Adam Reeve and Laurent Goujon have been invited
+to become committers. No new members have joined the Project Management
+Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Columnar format
+
+## Arrow Flight RPC notes
+
+## C++ notes
+
+### Acero
+
+### Compute
```

Comment:
@anjakefala Do you want to add a note about the ChunkResolver API change?

@pitrou do any other C++ changes look like they should be mentioned here?

Reply:
> @pitrou do any other C++ changes look like they should be mentioned here?

I don't think so, no.

Reply:
I added a note about ChunkResolver so this section is now done.